R/init.R

```diff
@@ -45,14 +42,9 @@
 #'
 #' @examples
 #' app <- init(
-#'   data = teal_data(
-#'     new_iris = transform(iris, id = seq_len(nrow(iris))),
-#'     new_mtcars = transform(mtcars, id = seq_len(nrow(mtcars))),
-#'     code = "
-#'       new_iris <- transform(iris, id = seq_len(nrow(iris)))
-#'       new_mtcars <- transform(mtcars, id = seq_len(nrow(mtcars)))
-#'     "
-#'   ),
+#'   data = teal_data() |>
+#'     within(new_iris <- transform(iris, id = seq_len(nrow(iris)))) |>
+#'     within(new_mtcars <- transform(mtcars, id = seq_len(nrow(mtcars)))),
```

Comment:
Maybe just one `within` and multiple lines of code included in `{}`?

Reply:
yup - applied your suggestion